src/GameLogic/PlugIns/CreateItemChatCommandPlugIn.cs

```diff
@@ -0,0 +1,146 @@
+                player.ViewPlugIns.GetPlugIn<IShowMessagePlugIn>()?.ShowMessage(e.Message, MessageType.BlueNormal);
+            }
+        }
+
+        private static Item CreateItem(Player player, Arguments arguments)
+        {
+            var item = new TemporaryItem();
+            var itemDefinition = player.GameContext.Configuration.Items.FirstOrDefault(def => def.Group == arguments.Group && def.Number == arguments.Number);
+            if (itemDefinition == null)
+            {
+                throw new ArgumentException($"[GM][/item] {arguments.Group} {arguments.Number} does not exists");
+            }
+            item.Definition = itemDefinition;
+            item.Level = arguments.Level;
+            item.Durability = itemDefinition.Durability;
+            item.HasSkill = arguments.Skill;
+            
+            if (arguments.Opt > 0)
+            {
+                var optionLink = new ItemOptionLink
+                {
+                    ItemOption = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                        .First(o => o.OptionType == ItemOptionTypes.Option),
+                    Level = arguments.Opt
+                };
+                item.ItemOptions.Add(optionLink);
+            }
+
+            if (arguments.Luck)
+            {
+                var optionLink = new ItemOptionLink
+                {
+                    ItemOption = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                        .First(o => o.OptionType == ItemOptionTypes.Luck)
+                };
+                item.ItemOptions.Add(optionLink);
+            }
+
+            if (arguments.Exc > 0)
+            {
+                var excellentOptions = item.Definition.PossibleItemOptions.SelectMany(o => o.PossibleOptions)
+                    .Where(o => o.OptionType == ItemOptionTypes.Excellent)
+                    .Take(arguments.Exc);
+                
+                excellentOptions.ForEach(option =>
+                {
+                    var optionLink = new ItemOptionLink()
+                    {
+                        ItemOption = option
+                    };
+                    item.ItemOptions.Add(optionLink);
+                });
+            }
+
+            return item;
+        }
+
+        private static Arguments MarshallArguments(List<string> argumentsList)
+        {
+            if (argumentsList.ElementAtOrDefault(0) == null || argumentsList.ElementAtOrDefault(1) == null)
+            {
+                throw new ArgumentException(Usage);
+            }
+
+            return new Arguments()
+            {
+                Group = byte.Parse(argumentsList.ElementAt(0)),
+                Number = short.Parse(argumentsList.ElementAt(1)),
+                Level = byte.Parse(argumentsList.ElementAtOrDefault(2) ?? "0"),
+                Exc = byte.Parse(argumentsList.ElementAtOrDefault(3) ?? "0"),
+                // Every excellent item has Skill
```

Comment:
Every excellent item has Skill, but only if a Skill is available in the ItemDefinition